src/autorest-core/test/resources/literate-example/csharp/Models/Error.cs

```diff
@@ -0,0 +1,48 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
+// Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
why are these files (`src/autorest-core/test/resources/literate-example/csharp/`) in this PR?